.circleci/config.yml

```diff
@@ -51,14 +51,14 @@ workflows:
       - build-only:
           matrix:
             parameters:
-              architecture: [amd64, arm64]
```

Comment:
maybe add a comment like TODO: we should add arm64 below, but we currently have troubles publishing multi-arch docker images